numba_dppy/config.py

```diff
@@ -97,4 +98,5 @@ def _readenv(): ...
 NATIVE_FP_ATOMICS = _readenv("NUMBA_DPPY_ACTIVATE_ATOMCIS_FP_NATIVE", int, 0)
 LLVM_SPIRV_ROOT = _readenv("NUMBA_DPPY_LLVM_SPIRV_ROOT", str, "")
 # Emit debug info
-DEBUG = os.environ.get("NUMBA_DPPY_DEBUG", None)
+DEBUG = _readenv("NUMBA_DPPY_DEBUG", int, 0)
+DEBUGINFO = _readenv("NUMBA_DPPY_DEBUGINFO", int, DEBUGINFO_DEFAULT)
```

Comment:
Why do we need special environment variable for dppy? Why not use numba default?